SpecFlowProjectMars1/Hooks/Hooks.cs

```diff
@@ -0,0 +1,137 @@
+﻿using OpenQA.Selenium;
+using OpenQA.Selenium.Chrome;
+using OpenQA.Selenium.Support.UI;
+using SeleniumExtras.WaitHelpers;
+using SpecFlowProjectMars1.Pages;
+using SpecFlowProjectMars1.Utilities;
+using TechTalk.SpecFlow;
+
+namespace SpecFlowProjectMars1.Hooks
+{
+
+    [Binding]
+    public class Hooks: CommonDriver
+    {
+
+       
+        [BeforeScenario("language")]
+        public void BeforeScenarioLanguage()
+        {
+
+            LoginActions loginPageobj = new LoginActions();
+            loginPageobj.Login("test123456@test.com", "test123456");
+            ClearLanguageTestData();
+        }
+        [BeforeScenario("skill")]
+        public void BeforeScenarioSkill()
+        {
+            LoginActions loginPageobj = new LoginActions();
+            loginPageobj.Login("test123456@test.com", "test123456");
+            ClearSkillTestData();
```

Comment:
this should not be in this hook but in another hook [BeforeTestRun] [AfterTestRun]
 As the name suggests “BeforeTestRun” will trigger before the test run and “AfterTestRun” will trigger after complete test run.look at https://codebun.com/how-to-implement-hooks-in-specflow-c/